package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "bootstrap": "^4.5.2",
```

Comment:
I think we don't need this anymore because we added `Link` to `index.html`. Try to remove this line from here and `yarn install` again to update `yarn.lock`.